sdks/java/io/google-cloud-platform/build.gradle

```diff
@@ -18,7 +18,13 @@
 
 import groovy.json.JsonOutput
 
-plugins { id 'org.apache.beam.module' }
+plugins {
+  id 'org.apache.beam.module'
+  // For resolving Cloud Bigtable dependencies from custom registry that includes Change Stream API.
+  id "maven-publish"
```

Comment:
maven-publish is handled in BeamModulePlugin and to my understanding should not be added here.

move the below 
```
repositories {
  maven {
```
clause to https://github.com/apache/beam/blob/master/buildSrc/src/main/groovy/org/apache/beam/gradle/Repositories.groovy is the right approach

Reply:
The complication, and I couldn't figure this out, is the URL is `artifactregistry` which requires `com.google.cloud.artifactregistry.gradle-plugin` plugin. It's not obvious how to include the plugin in Repositories.groovy.